requirements.txt

```diff
@@ -1,6 +1,7 @@
 Flask==0.10.1
 uWSGI==2.0.12
 Flask-SQLAlchemy==2.1
+PyMySQL==0.7.6
```

Comment:
Happy to see the python only driver being used! Let's hope performance never forces the project to MySQLdb.